api/src/main/java/api/UserAnswerResource.java

```diff
@@ -52,4 +53,28 @@
     @DELETE
     @Path("me/answers/{answerId}")
     Observable<Void> deleteAnswer(Auth auth, @PathParam("answerId") long answerId);
+
+    /**
+     * Upvotes (+1) an answer.
+     * Upvoting an existing downvote will result in a neutral (0) vote.
+     * Upvoting an existing upvote will result in error.
+     * @param auth
+     * @param answerId
+     * @return
+     */
+    @PATCH
+    @Path("me/answers/upvote/{answerId}")
```

Comment:
more of an discussion, but should it be `me/answers/{answerId}/upvote/`? if we se answers as an collection

Reply:
That was actually how I did it first, but then I saw how "accept" was done and followed that pattern.

Reply:
I will swap back.